src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/ProjectHomeView.kt

```diff
@@ -0,0 +1,65 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projecthome
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Pos
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
+import org.wycliffeassociates.otter.jvm.app.widgets.projectcard
+import tornadofx.*
+
+class ProjectHomeView : View() {
+
+    val viewModel: ProjectHomeViewModel by inject()
+    override val root = borderpane {
+        top = hbox { alignment = Pos.CENTER_RIGHT
+            button("Refresh") {
+                prefWidth = 232.0
+                prefHeight = 40.0
+                action {
+                    viewModel.getAllProjects()
+                }
+            }}
+        style {
+            setPrefSize(1200.0, 800.0)
+        }
+        center = datagrid(viewModel.allProjectsProperty.value) {
```

Comment:
if we change allProjects to just be an observablelist instead of a property, we can do datagrid(viewModel.allProjects) instead. Wrapping in properties is good for strings, booleans, singular objects, etc., but observable lists should be fine for lists

Reply:
yes, but the bind delegate from tornadofx uses properties by default does it not?

Reply:
True, but with an observable list you won't need to use bind actually. You can just do `val allProjects = model.allProjects` (or similar) in the view model. We can already bind directly to an observable list. We were making custom properties for other fields because they were not observable by default.

Reply:
yes you're right. Problem i'm having right now is i'm not actually initializing a model in the view model. i'm using some ItemViewModel from tornadofx, but i suppose i can go ahead and change it. might have to do that for the wizard too but i'll go give it a shot real quick

Reply:
Could you do something like `val allProjects = item.allProjects`? I think item is property in ItemViewModel